content/javascript/concepts/objects/terms/getOwnPropertyDescriptors/getOwnPropertyDescriptors.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: '.getOwnPropertyDescriptors'
+Description: 'returns all own property descriptors of a given object'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+  - 'Web Applications'
+Tags:
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/create-a-back-end-app-with-javascript'
+---
+
+The **`Object.getOwnPropertyDescriptors()`** method returns all own property descriptors of a given object. It is useful for extracting property descriptors, including configurable, enumerable, value, and writable properties, from an object.manipulation.
+
+## Syntax
+
+```pseudo
+Object.getOwnPropertyDescriptors(obj)
+```
+
+`obj` is the given obeject.
+
+## Examples
```

Comment:
Remove this heading and add the rest of the two (`Example 1` & `Example 2`) headings as h2.